USAGE.md

```diff
@@ -32,19 +50,23 @@ terminal.
 - On Linux, your distribution will provide a terminal; it can generally be opened with
 Ctrl+Shift+T or by searching for "Terminal" in your application menu.
 
-### On Docker and Non-Free Operating Systems ###
+### On OED and Non-UNIX Operating Systems ###
 
-Docker is well-supported on most free/libre operating systems, including GNU/Linux,
-FreeBSD, OpenBSD, and Plan9.
+Windows is not a supported platform. You can probably run OED there, but you will
+experience problems.
 
-Docker is not supported on Windows. Windows is incapable of supporting LXE containers,
-the technology on which Docker is based, and Microsoft
-[artifically limits](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/)
-the use of Docker for Windows, a virtualized technology.
+Docker is a Linux technology. It is based on Linux kernel features including namespaces
+and cgroups. On Mac OS, you can use [Docker for Mac](https://docs.docker.com/docker-for-mac/)
+to run OED in Docker. Microsoft [artifically limits](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/)
```

Comment:
The term "artifically limits" (correct spelling of artificially) does not seem a neutral term. From what I understand, it does not run on the home version of Windows but does run on others. Is that correct for OED? If so, should we state that or link to MS and say they specify which versions of the OS are supported?

Reply:
> The term "artifically limits" (correct spelling of artificially) does not seem a neutral term. From what I understand, it does not run on the home version of Windows but does run on others. Is that correct for OED?

This is precisely what I mean by an "artificial" limit. They have the capacity to support it on all versions of Windows and choose not to do so. That link explains all the various complexity inherent therein. I can change it to make that more clear.

Reply:
I think we should try to stay neutral about what we say about OSes. I might prefer something similar to: "Docker is not supported on all versions of Windows. You should see [link] for details from Microsoft. You will need to manually install OED ([link to directions?] if this is the case for you." How does this sound?

Reply:
I'm somewhat reluctant to do so given that this _is in fact_ a bad faith action by Microsoft Corporation. They previously allowed users to do something and then just _stopped_ letting them, almost totally invalidating an entire semester's worth of work in the process. I don't feel like there's a particularly good reason for us to be neutral about that.

I'll to take the language out, but given that I've spent a significant portion of my time this year trying (and often failing) to get other students set up with a working development environment on Windows, I think it might be a good idea to have a conversation, as a team, about whether or not we actually want to make an effort to support that platform.

As much as we may want to be neutral, the fact remains that people on Linux and Mac OS X are able to get set up with the project in a few minutes, while it takes hours at best on Windows. The whole point of Dockerization was to solve that, and it did, until Microsoft decided to hold that feature for ransom.